.babelrc

```diff
@@ -1,3 +1,3 @@
 {
- "presets": ["es2015", "react", "stage-0"]
+ "presets": ["es2015", "react", "stage-1"]
```

Comment:
`Thanks for using Babel: we recommend using babel-preset-env`

"presets": ["env", "react"]